spec/features/tagging_content_spec.rb

```diff
@@ -1,6 +1,15 @@
 require "rails_helper"
 
-RSpec.describe "Tagging content" do
+RSpec.describe "Tagging content", type: :feature do
+  include PublishingApiHelper
+
+  before do
```

Comment:
I agree with the problem but not entirely sure how to fix it. Option 2 sounds the best - I'd ideally like some kind of factory that will default the things I don't care about. Is there some existing example of this I can look at?

What do you think about something like this:

```ruby
  def publishing_api_has_organisations(base_paths)
    publishing_api_has_linkables(
      stubbed_organisations.select { |org| base_paths.include?(org["base_path"]) },
      document_type: 'organisation'
    )
  end
```

```ruby
    publishing_api_has_organisations(
      [
        "/government/organisations/student-loans-company",
        "/my-content-item",
      ]
    )
```